src/actions/acquire_single_freq_fft.py

```diff
@@ -179,9 +178,13 @@ def __call__(self, schedule_entry_name, task_id):
 
         self.test_required_components()
         self.configure_sdr()
+        start_time = utils.get_datetime_str_now()
         data = self.acquire_data()
+        end_time = utils.get_datetime_str_now()
```

Comment:
If these times are for the capture, we may consider pushing them down to `usrp_interface` right around `usrp_recv_num_samps`. If there's an overflow error, data will be retaken to try and get the right number, so these may not correspond to the true start/stop times. This may be better to do in the `radio_interface` branch though.

Reply:
These are used for the measurement object `time_start` and `time_stop`. Do you think they should be pushed to the `usrp_interface`?

Reply:
I think you're right and the capture `datetime` should be the one moved. I think these can stay here.